daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -364,6 +434,9 @@ func ValidateLogOpt(cfg map[string]string) error {
 	if cfg[logGroupKey] == "" {
 		return fmt.Errorf("must specify a value for log opt '%s'", logGroupKey)
 	}
+	if cfg[logCreateGroupKey] != "" && cfg[logCreateGroupKey] != "true" && cfg[logCreateGroupKey] != "false" {
```

Comment:
Should we use [`strconv.ParseBool()`](https://golang.org/pkg/strconv/#ParseBool) here?